.gitignore

```diff
@@ -36,6 +36,9 @@ build/Release
 
 # Dependency directories
 node_modules/
+.yarn/*
```

Comment:
It might be good to group together all Yarn-related things in this file, so it's more easily copy-and-pasteable.

Reply:
Also, this doesn't seem to include the full list here, any reason? https://yarnpkg.com/getting-started/qa#which-files-should-be-gitignored

Reply:
There weren't any folders or files for the other things listed there, but I'll add them for good measure.